docs/src/lib/functions.md

```diff
@@ -8,87 +8,116 @@ CurrentModule = DataFrames
 Pages = ["functions.md"]
 ```
 
-## Joining, Grouping, and Split-Apply-Combine
+## Constructing data frames
+```@docs
+copy
+DataFrame!
```

Comment:
It feels weird not to have this at the same place as constructors.

Reply:
Actually the problem is that I would put `DataFrame` constructor here, but we had to remove it unfortunately, because Documenter.jl disallows it (as `DataFrame` is at the same a type and it already gets included in the `Types` tab).

So given this restriction - what do you think is best to do?

Reply:
Would it make sense to put `DataFrame!` in the Types section? Otherwise it's not a big deal to have it here.

Reply:
I will move it there